rpc/params/secrets.go

```diff
@@ -291,6 +291,14 @@ type GrantRevokeSecretArg struct {
 	Role string `json:"role"`
 }
 
+// GrantRevokeUserSecretArg holds the args for changing access to a user secret.
+type GrantRevokeUserSecretArg struct {
```

Comment:
Can we call this `GrantRevokeUserSecretArg`

Reply:
Are they same?